docs/paper/admin/how-to/basic-troubleshooting.md

```diff
@@ -0,0 +1,135 @@
+
+The thread dump from Paper Watchdog can be found below. If you find any plugin's name in there, talk to the plugin's author about that, not PaperMC.
+
+### Case 02: Stack Trace
+
+Almost every problem you encounter will print error message lines, which are called "stack trace", on the server console. Examining the stack trace will help you find out what is causing problems on your server.
+
+The stack trace starts with the error message, exception type, and exception message.
+
+Both error messages and exception messages were put there by the developer of either your plugin or paper server. These messages tell you what problem your server experienced.  
+An exception type like `java.lang.RuntimeException` tells you the type of the exception. This will help the developer (and you) understand the type of problem.
+
+Many lines beginning with `at` may appear beneath the exception message. These are the bodies of the stack trace. These lines tell you where the problem starts. The top line of the body of the stack trace will tell you exactly where the problem occurred, as well as the name of the JAR file.
+
+If you find any plugin's name in the stack trace, head to [Check Plugin Updates](#check-plugin-updates) and read from there. In most cases, the plugin, whose name is found on the stack trace, is causing the problem. If not, continue reading.
+
+Here are some examples of stack trace.
+
+<details>
+  <summary>Example 01: Server attempted to load chunk saved with newer version of minecraft!</summary>
+
+```plaintext
+[00:00:00 WARN]: java.lang.RuntimeException: Server attempted to load chunk saved with newer version of minecraft! 3218 > 3120
+```
+
+You tried to load the world generated with a higher version of Minecraft. You cannot do this.  
+If you don't have any backup of your world before the chunk version update, you must use your updated world with a higher version of Minecraft.
+
+</details>
+
+<!-- Another examples -->
+
+## Find Out What Plugin Is Causing The Problem
+
+If you can't find the name of any plugin in the thread dump or stack trace, try these steps.
+
+### Disable All Plugins
+
+To determine if it is a plugin or the paper itself that is causing problems, disable all of your plugins first.
+
+You can disable all of your plugins by renaming the `plugins` directory to something else, such as `plugins-disabled`, or by archiving the `plugins` directory and deleting it.
+
+### Run Server Without Any Plugins
+
+Try to run your server without any plugins. If you rename or delete `plugins` directory, your server will run without any plugins.
+
+If you don't have any issues with this condition, it's the plugin that's causing them.  
+If you still experience problems, head to [Paper Documentation](#paper-documentation) because it might be paper that is creating problems, and most of them are because of a misconfigured paper server.
```

Comment:
“because it may be that your Paper server is mis-configured and this is creating issues. “